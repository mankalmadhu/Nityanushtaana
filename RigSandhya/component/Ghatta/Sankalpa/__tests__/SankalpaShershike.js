import React from 'react';
import ಸ೦ಕಲ್ಪಶೀರ್ಷಿಕೆ from '../SankalpaShershike';
import Renderer from 'react-test-renderer';



jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');

test("ಸ೦ಕಲ್ಪಶೀರ್ಷಿಕೆ", () => {

  expect(Renderer.create(
    <ಸ೦ಕಲ್ಪಶೀರ್ಷಿಕೆ />
  ).toJSON()).toMatchSnapshot();

});