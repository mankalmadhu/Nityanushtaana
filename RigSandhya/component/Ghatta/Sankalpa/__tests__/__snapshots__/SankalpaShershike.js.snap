// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಸ೦ಕಲ್ಪಶೀರ್ಷಿಕೆ 1`] = `
<ವಿಷಯ
  ವಿನ್ಯಾಸ="ಶೀರ್ಷಿಕೆ"
>
  ಸ೦ಕಲ್ಪ
</ವಿಷಯ>
`;
